### PR TITLE
Added an FAQ

### LINKED ISSUE #72: Create FAQ page
- Answers for FAQs
  - [ ] What is $ustainOSS?
  - [ ] How is it written? (Sustain OSS, $ustainoss, $ustainOSS, $ustain OSS)
  - [ ] How many people will be there?
  - [ ] What is the price?
  - [ ] Will you have discounts for students?
  - [ ] What will you be serving for lunch?
  - [ ] Will there be talks?
  - [ ] How many days is it?
  - [ ] Will there be hotel accomidations?
  - [ ] Will there be other events in different countries?
  - [ ] Do you accept in-kind sponsorships?
  - [ ] When will I receive a ticket?
  - [ ] What other events can you compare $ustain OSS to?

### PR DESCRIPTION
**Pull request by RichardLitt**
Closes #72

### CODE REVIEW

**Comment by whit537** [comment]
Ping @jdorfman?

File: faq.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+slug: faq
+---
+
+## What is $ustainOSS?
+
+A one-day conference and discussion, were we'll talk about how to sustain open source software. It's not a large conference with talks and keynotes, and it's not an unconference; it's a gather of open source contributors, maintainers, and sustainers; industry leaders and people interested in open source; and friends. The goal is to understand the long term viability of open source as a model for producing software.
+
+## How is it written? (Sustain OSS, $ustainoss, $ustainOSS, $ustain OSS)
+
+SustainOSS is fine. But anything really works.
```
Comment (jdorfman) [inline faq.md]: The itself is "$ustain". The domain and twitter is `sustainoss` so it is more descriptive plus it is impossible to acquire the domain and twitter account.

File: faq.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+## How many people will be there?
+
+We expect to have somewhere around 50 people. We'd like to have more. Why don't you sign up?
```
Comment (jdorfman) [inline faq.md]: link to signup form

File: faq.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+## What is the price?
+
+The regular price is $50. This goes towards the venue, running costs, food, and to the costs of organizing it. We'd appreciate if you paid $100 and allowed someone else to come; if you can't pay $50, there is also a free tier. Finally, if you're a sponsor, a $1000 ticket is available, and we'd feature your logo on the website and mention you at the conference as a sponsor. Everybody wins.
+
+## Will you have discounts for students?
+
+We hope so! Keep your eyes peeled; we're working on getting sponsorships for studnets.
```
Comment (jdorfman) [inline faq.md]: Just mention the free tickets, grab them while they are available.

File: faq.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+## What will you be serving for lunch?
+
+More on this later. We'll cater to vegetarians; paleo eaters may have to go hunting and gathering during the afternoon sessions.
```
Comment (jdorfman) [inline faq.md]: we need to check with Brandon/Nadia

**Comment by jdorfman** [comment]
great job @RichardLitt

**Pull request merged**